examples/js/postprocessing/CubeTexturePass.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @author bhouston / http://clara.io/
+ */
+
+THREE.CubeTexturePass = function ( scene, camera, envMap, opacity ) {
+
+	THREE.Pass.call( this );
+
+	this.scene = scene;
```

Comment:
I see, I see. It would be nice to add a comment, otherwise someone will try to clean it up in the future.